lib/flatpak-installation.c

```diff
@@ -1396,6 +1418,14 @@ flatpak_installation_install_full (FlatpakInstallation    *self,
  *
  * Install a new application or runtime.
  *
+ * Note that this function was originally written to always return a
+ * #FlatpakInstalledRef. Since 0.9.12.12, passing
```

Comment:
0.9.12.12

Reply:
I'm a bit confused, this already says 0.9.12.12 ? (unless I missed something?)

Reply:
0.9.12.12 isn’t a released version. 0.9.12 is a valid version number (and has already been released). You probably want to say 0.9.13.

Reply:
Got it, thanks